cmd/troubleshoot/dynakube_test.go

```diff
@@ -349,10 +358,12 @@ func (builder *testDynaKubeBuilder) withCloudNativeFullStackCustomImage(image st
 func (builder *testDynaKubeBuilder) withHostMonitoringCustomImage(image string) *testDynaKubeBuilder {
 	if builder.dynakube.Spec.OneAgent.HostMonitoring != nil {
 		builder.dynakube.Spec.OneAgent.HostMonitoring.Image = image
+		builder.dynakube.Status.OneAgent.ImageID = image
```

Comment:
from what I can see you can do this line out of if/else. cause we do same in both branches. 🤔